Remove PaintWorklet as subclass of Worklet (#25996)
mdn/content: files/en-us/web/api/worklet/index.md

  </tbody>
</table>

> > **Note:** Paint worklets, defined by the [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API), don't subclass {{domxref("Worklet")}}. They are used for programmatically generating an image where a CSS property expects a file. A paint worklet is accessed throught a regulare `Worklet` obtained using {{DOMxRef("CSS.paintWorklet")}}.

Reviewer: ```suggestion
> **Note:** Paint worklets, defined by the [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API), don't subclass {{domxref("Worklet")}}. They are used for programmatically generating an image where a CSS property expects a file. A paint worklet is accessed through a regular `Worklet` obtained using {{DOMxRef("CSS.paintWorklet")}}.
```

But also the middle sentence here sits oddly in this paragraph (as if the fact that "They are used for programmatically generating an image..." is connected to the fact that it doesn't have its own `Worklet` subclass. Switching them flows better:

> **Note:** Paint worklets are used for programmatically generating an image where a CSS property expects a file. Paint worklets don't subclass {{domxref("Worklet")}}: instead, a paint worklet is accessed through a regular `Worklet` obtained using {{DOMxRef("CSS.paintWorklet")}}.

...or just deleting the sentence "They are used for programmatically generating an image..." would be OK, or perhaps better, IMO. (I know that makes this para inconsistent with the table rows, and I think that's not a problem).

Author: ```suggestion
> **Note:** Paint worklets, defined by the [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API), don't subclass {{domxref("Worklet")}}. They are accessed through a regular `Worklet` object obtained using {{DOMxRef("CSS.paintWorklet")}}.
```
